gradle/libs.versions.toml

```diff
@@ -1,12 +1,12 @@
 [versions]
-kotlin = "1.8.0"
+kotlin = "1.8.22"
 
-minecraft = "1.19.3"
-qfapi = "5.0.0-alpha.9+0.72.0-1.19.3"
-qkl = "1.1.0+kt.1.8.0+flk.1.9.0"
-loader = "0.17.8"
-mappings = "1.19.3+build.9"
-loom = "1.+"
+minecraft = "1.20.1"
+qfapi = "7.0.3+0.83.1-1.20.1"
+qkl = "2.1.0+kt.1.8.22+flk.1.9.4"
+loader = "0.19.1"
+mappings = "1.20.1+build.1"
+loom = "1.2.3"
```

Comment:
thanks this is very smart we should do this more

Reply:
:3